examples/julia_logo.jl

```diff
@@ -0,0 +1,159 @@
+using Javis
+using DelimitedFiles
+using OffsetArrays, FFTViews # (not strictly necessary, but make DFT indexing more intuitive)
+
+# Import the discrete fourier transform of our points and recast them in their original complex form
+ps = readdlm("julia_logo.csv", ',')
+cs = complex.(ps[:, 1], ps[:, 2]) |> FFTView
+
+vid = Video(800, 500)
+
+function ground(args...)
+    background("black")
+    sethue("black")
+    circle(Point(0, 0), 200, :fill)
+end
+
+function object(
+    p = O,
+    color = "white";
+    size = 25,
+    edge = "solid",
+    style = :stroke,
+    opacity = 1 / 3,
+)
+    sethue(color)
+    setdash(edge)
+    setopacity(opacity)
+    circle(p, size, style)
+    return p
+end
+
+function draw_line(
+    p1 = O,
+    p2 = O;
+    color = "white",
+    action = :stroke,
+    edge = "solid",
+    linewidth = 3,
+)
+    sethue(color)
+    setdash(edge)
+    setline(linewidth)
+    line(p1, p2, action)
+end
+
+function draw_path!(path, pos, color)
+    sethue(color)
+    push!(path, pos)
+    draw_line.(path[2:end], path[1:(end - 1)]; color = color)
+end
+
+import Javis.Point
+Point(c::Complex) = Point(real(c), imag(c))
+distance(p1::Point, p2::Point) = sqrt((p2.x - p1.x)^2 + (p2.y - p1.y)^2)
+
+# Draw a vector between points p1 and p2
+function vector(p1, p2; color = "white", opacity = 1 / 2, linewidth = 1, tipsize = 0.1)
+    sethue(color)
+    setopacity(opacity)
+    setline(linewidth)
+
+    θ = (p2.y - p1.y) / (p2.x - p1.x) |> atan # angle b/w p1 and p2
+    s = sign(p2.x - p1.x)
+
+    α = tipsize                             # heigth of the tip triangle (in terms of the distance b/w p1 and p2)
+    ϑ = α / (1 - α) / 2 |> atan                 # angle b/w p1 and "bottom-left" vertex of the tip triangle
+    d = distance(p1, p2)
+    x = d * (1 - α) / cos(ϑ)                    # distance b/w p1 and triangle vertices
+
+    v1 = p1 + s * x * Point(exp((θ + ϑ) * im))    # first triangle vertex
+    v2 = p1 + s * x * Point(exp((θ - ϑ) * im))    # second triangle vertex
+
+    line(p1, p2, :stroke)                   # vector line
+    poly([p2, v1, v2, p2], :fill)           # vector tip
+end
+
+function draw_logo(n::Int; frames = 2160)
+    # TODO: (1) Fix indices so that I don't need to go to end-1 (a problem for very small n)
+
+    base_size = 7.5e-5
+    n_rotations = 2         # how many times the "tip" goes around the drawing
+
+    remap_idx(i::Int) = (-1)^(i - 1) * floor(Int, (i + 1) / 2)
+    remap_inv(n::Int) = 2n * sign(n) - 1 * (n > 0)
+
+    n0, nT = -Int((n - 1) / 2), Int((n - 1) / 2)
```

Comment:
fails for odd `n`

Reply:
Ah, you're right. I can add a check that makes sure n is always odd (not sure how things would change with an even number of points, but can check that too)

Reply:
Yeah or ceil the division maybe